src/Vasont.Inspire.Core.csproj

```diff
@@ -67,10 +67,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="PdfLibCore" Version="2.3.0" />
-    <PackageReference Include="Svg" Version="3.2.3" />
-    <PackageReference Include="System.Drawing.Common" Version="5.0.2" />
-    <PackageReference Include="System.Resources.Extensions" Version="5.0.0" />
+    <PackageReference Include="PdfLibCore" Version="2.5.0" />
+    <PackageReference Include="Svg" Version="3.4.7" />
+    <PackageReference Include="System.Drawing.Common" Version="9.0.3" />
+    <PackageReference Include="System.Resources.Extensions" Version="9.0.3" />
```

Comment:
Make sure this doesn't depend on NET9, to avoid errors in our NET8 code. We should also be moving away from this package as well, but slow going as you know.

Reply:
Double checked both of these and they support multiple versions 
![image](https://github.com/user-attachments/assets/8922e04f-c6f2-48a2-afc3-03676bfbd667)